app/assets/javascripts/item_selector/item_selector_ad_hoc_items.js

```diff
@@ -5,12 +5,12 @@ var itemSelectorAdHocItems = (function() {
     selector: '[data-behavior="ad-hoc-items"]'
   };
 
-  var santizeCallnumber = function(callnumber) {
+  var sanitizeValue = function(callnumber) {
```

Comment:
ah - good call.  I think I did that at one point and lost the commit.  will fix anon.